sdk/media/azure-media-analytics-edge/samples/sample_lva.py

```diff
@@ -0,0 +1,86 @@
+
+import json
+import os 
+#from azure.media.livevideoanalytics.edge._generated.models import *
+from azure.media.analyticsedge import *
+from azure.iot.hub import IoTHubRegistryManager
```

Comment:
We should add a comment about what package people need to pip install to get that import to work

Reply:
Added an inline comment and one in the README.md as well